InfiniteScrollView.js

```diff
@@ -39,8 +51,15 @@ class InfiniteScrollView {
   }
 
   render() {
-    if (this.props.isLoadingMore) {
-      var loadingIndicator = React.cloneElement(
+    let statusIndicator;
+
+    if (this.state.isDisplayingError) {
+      statusIndicator = React.cloneElement(
+        this.props.renderLoadingErrorIndicator(this._retryOnLoadMore.bind(this)),
```

Comment:
I'm thinking it might be a bit better to pass in props instead of a function, since that's what we do with a lot of the other `renderX` props. Ex: `{ onRetryLoadMore: ... }` (I called it onRetryLoadMore since the handlers are prefixed with "on".)